config.json

```diff
@@ -622,7 +622,13 @@
       "core": false,
       "difficulty": 6,
       "slug": "robot-simulator",
-      "topics": null,
+      "topics": [
+        "classes",
+        "algorithms",
+        "logic",
+        "loops",
+        "enumerations"
```

Comment:
Great suggestions! :) The only one I'm not sure about is `algorithms`. You could argue that you're using an algorithm to solve this exercise, but then you could make that argument for every exercise and I'm not sure it would be very useful to add it as a topic to every exercise. So I think it's best to reserve `algorithms` for exercises that require you to implement a specific algorithm, like `luhn` for example. Does that make sense? Do you agree?